### PR TITLE
added Footer

### PR DESCRIPTION
**Pull request by christopheluc62**

### CODE REVIEW

**Review by Hacene010 — DISMISSED** [review]

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -1,4 +1,54 @@
...
+        >
+          General Conditions of Use
+        </a>
+        <a
+          href="https://www.wildcodeschool.com/fr-FR/charte-donnees-personnelles"
+          id="link"
+        >
+          Personal Data Charter
+        </a>
+        <h3>©2021 WILD & SONS. ALL RIGHTS RESERVED</h3>
```
Comment (LoicBrassart) [inline src/components/Footer.jsx:49]: Heh, nice touch ;-)

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -1,4 +1,54 @@
...
+      <div className="infos">
+        <a
+          href="https://www.wildcodeschool.com/fr-FR/condition-generales-utilisation"
+          id="link"
+        >
+          General Conditions of Use
+        </a>
+        <a
+          href="https://www.wildcodeschool.com/fr-FR/charte-donnees-personnelles"
+          id="link"
```
Comment (LoicBrassart) [inline src/components/Footer.jsx]: That's a no-go ! The `id` attribute should be unique on a page :x:

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
There's a "big" HTML error to fix, but it should be quick and easy ;-)

File: src/components/Footer.jsx (hunk 2 above)
Comment (christopheluc62) [reply src/components/Footer.jsx]: id replaced by className

**Review by Hacene010 — DISMISSED** [review]

**Comment by LoicBrassart** [comment]
Naaaaaah !
You assign your SM, and *your SM* assigns me :p

**Comment by Hacene010** [comment]
> Naaaaaah !
> You assign your SM, and _your SM_ assigns me :p

**Comment by Hacene010** [comment]
My SM is the author of the PR. that's why I assigned you directly :)

**Review by Hacene010 — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
No More Evil! \o/

**Pull request merged**